src/components/user-menu/user-menu.tsx

```diff
@@ -0,0 +1,71 @@
+import { NavLink, useNavigate } from '@remix-run/react';
+import { useMemberInfo } from '~/lib/hooks';
+import { Avatar } from '../avatar/avatar';
+import {
+    DropdownMenu,
+    DropdownMenuItem,
+    DropdownMenuSeparator,
+} from '../dropdown-menu/dropdown-menu';
+import { ChevronDownIcon } from '../icons';
+
+import styles from './user-menu.module.scss';
+
+export const UserMenu = () => {
+    const { isLoggedIn, member } = useMemberInfo();
+
+    const navigate = useNavigate();
+
+    const handleRootClick = () => {
```

Comment:
I believe not using `DropdownMenu` component at all if user is not logged in would simplify this component a lot

```
if (!isLoggedIn) {
    return <div onClick={() => navigate('/login')}>Log In</div>
} else {
    return <DropdownMenu />
}
```

Reply:
simplified layout